keras_cv/layers/object_detection_3d/voxel_utils.py

```diff
@@ -258,8 +213,8 @@ def combined_static_and_dynamic_shape(
     Returns:
       A list of size tensor.shape.ndims containing integers or a scalar tensor.
     """
-    static_tensor_shape = tensor.shape.as_list()
-    dynamic_tensor_shape = shape_int_compatible(tensor)
+    static_tensor_shape = list(tensor.shape)
```

Comment:
potentially this is already what `ops.shape` doing for the tf backend now? not totally sure though

Reply:
Ahh yes, since your change to Keras Core I think this method is really no longer useful. I'm removing it.